en/docs/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md

```diff
@@ -49,18 +49,18 @@ In order to disable self signup for a particular tenant, follow the steps below.
 
 1. Sign in to the management console ( `https://<HostName>:9443/carbon` ) as tenant admin.
 
-2. Navigate to the `/_system/governance/apimgt/applicationdata/sign-up-config.xml` file in the registry again and set the `<SelfSignUp><EnableSignup>` element to false and save the content.
-
-3. Click **Resident** under **Identity Providers** on the **Main** tab and expand the **Account Management Policies** tab.
+2. Click **Resident** under **Identity Providers** on the **Main** tab.
 
     ![Identity Provider Resident]({{base_path}}/assets/img/learn/idp-resident.png)
 
-    ![Account Management Policies]({{base_path}}/assets/img/learn/account-management-policies.png)
+3. Expand the **User Onboarding** tab and expand the **Self Registration** tab
```

Comment:
Let's reword this as `Expand the **Self Registration** tab under **User Onboarding**`